R/lump.R

```diff
@@ -1,5 +1,5 @@
 #' Lump together least/most common factor levels into "other"
-#'
+#' @description `fct_lump` lumps together least/most common factor levels into "other".
```

Comment:
I think it's a bit easier to read if you just document both functions here — that way you can write a single paragraph. (It is a bit more confusing when you look at the docs for `fct_lump_min()` but I think that trade off is worth it).

Also can you please use parens at the end of function names? (https://style.tidyverse.org/documentation.html#r-code)

Reply:
I have made changes that take into account these remarks

Regards